pkg/occlient/occlient.go

```diff
@@ -377,42 +391,127 @@ func (c *Client) GetImageStreamsNames(namespace string) ([]string, error) {
 	return names, nil
 }
 
-// GetExposedPorts retruns list of ContainerPorts that are exposed by given image
-func (c *Client) GetExposedPorts(imageName string, imageTag string) ([]corev1.ContainerPort, error) {
+func isTagInImageStream(is imagev1.ImageStream, imageTag string) bool {
```

Comment:
Comments need to be added for this function

Reply:
Ack